pubnub/event_engine/models/states.py

```diff
@@ -305,14 +329,30 @@ def subscription_restored(self, event: events.SubscriptionRestoredEvent, context
             context=self._context
         )
 
+    def unsubscribe_all(self, event: events.UnsubscribeAllEvent, context: PNContext) -> PNTransition:
```

Comment:
The instructions below repeat in many places. It would be great to avoid these duplications. Let's rework it in the future if it's significant amount of work at this time